packages/components/app/styles/components/tooltip.scss

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+//
+// TOOLTIP & TOOLTIP-BUTTON
+//
+
+@use "../mixins/focus-ring" as *;
+
+:root {
+  // Dimensions:
+  --token-tooltip-max-width: 280px;
+
+  // Focus highlight position
+  --token-tooltip-tab-focus-inset-vertical: -10px;
+  --token-tooltip-tab-focus-inset-horizontal: -16px;
+
+  // Colors:
+  --token-tooltip-surface-primary: var(--token-color-palette-neutral-700);
+}
```

Comment:
Note: I can add tokens to JSON as one of the final steps.

Reply:
@alex-ju @didoo Any feedback on the token names? I think the CSS is stable now so I can go ahead and add the JSON.

Reply:
In terms of naming, they seem in line with what we have.
I have my doubts about chaining them (e.g. `--token-tooltip-surface-primary: var(--token-color-palette-neutral-700);`) do we need that abstraction? or could we just use `--token-color-palette-neutral-700` or use the color straight away `--token-tooltip-surface-primary: #0c0c0e`?

Reply:
looking at the CSS I see:
```
color: var(--token-color-foreground-high-contrast);
background-color: var(--token-tooltip-surface-primary);
```
so in one case we're using a semantic color token (`foreground-high-contrast`) in the other a component-level color token (`tooltip-surface-primary`); we should stick to one approach (probably the component-level one, see suggestion above)